Content.Shared/Access/Systems/AccessReaderSystem.cs

```diff
@@ -100,17 +104,53 @@ public bool IsAllowed(EntityUid user, EntityUid target, AccessReaderComponent? r
         if (!reader.Enabled)
             return true;
 
+        var validIds = GetAllowedEntities(user, reader).ToList();
+        // User is in a box.
+        if (TryComp(user, out CardboardBoxComponent? cardboardBox) && cardboardBox.Mover.HasValue)
+            user = cardboardBox.Mover.Value;
+        // Borg check.
+        if (TryComp(user, out BorgChassisComponent? borg))
```

Comment:
This also seems suspect so if you can think of another way I'd do that... It is a very specific situation so maybe its fine? I'd ask in the discord 🤷